src/dotnet/commands/dotnet-run/RunCommand.cs

```diff
@@ -119,14 +121,15 @@ private void Initialize()
 
                 if (projectFiles.Length == 0)
                 {
+                    var project = "--project";
+
                     throw new InvalidOperationException(
-                        $"{LocalizableStrings.RunCommandInvalidOperationException1} {directory}." + Environment.NewLine +
-                        LocalizableStrings.RunCommandInvalidOperationException2);
+                        $"Couldn\'t find a project to run. Ensure a project exists in  {directory}, or pass the path to the project using {project}");
```

Comment:
I was hoping to get some opinions on introducing a loc const for this. In a couple of instances like this one, multi-line strings had been broken into several consts. Loc strings are not supposed to be programmatically concatenated. It makes translation difficult by taking pieces of text out of context and it increases the likelihood that the concatenated string will be wrong in some languages. 

So with that in mind, I inlined and combined the strings but was unsure of the process for introducing new loc consts at this stage, since they won't have matching values in the xlf / resx files.

@nguerrera @piotrpMSFT